### PR TITLE
IGNITE-12951 Update documents for migrated extensions

### PR DESCRIPTION
**Pull request by samaitra**

### CODE REVIEW

File: modules/flink-ext/README.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ interested in):
         ...
         <dependency>
             <groupId>org.apache.ignite</groupId>
-            <artifactId>ignite-flink</artifactId>
+            <artifactId>ignite-flink-ext</artifactId>
             <version>${ignite.version}</version>
```
Comment (dmagda) [inline modules/flink-ext/README.txt:24]: Should we replace the _ignite.version_ parameter with _ignite-flink-ext_ one as long as the version of an extension is different from the Ignite one.

Comment (samaitra) [reply modules/flink-ext/README.txt]: @dmagda thank you for reviewing the changes, I have updated the PR.

Comment (dmagda) [reply modules/flink-ext/README.txt]: @samaitra, sorry for a belated check. All looks good to me, please go ahead and merge the changes to the master.

Comment (samaitra) [reply modules/flink-ext/README.txt]: @dmagda thank you, I have merged the changes.

**Pull request closed without merging**